core/src/main/java/com/novoda/noplayer/exoplayer/CodecSelector.java

```diff
@@ -0,0 +1,47 @@
+package com.novoda.noplayer.exoplayer;
+
+import com.google.android.exoplayer2.mediacodec.MediaCodecInfo;
+import com.google.android.exoplayer2.mediacodec.MediaCodecSelector;
+import com.google.android.exoplayer2.mediacodec.MediaCodecUtil;
+
+/**
+ * CodecSelector that allows overriding of the requiresSecureDecoder to allow
+ * downgrading of security but not upgrading.
+ */
+class CodecSelector implements MediaCodecSelector {
+
+    private final InternalMediaCodecUtil internalMediaCodecUtil;
+    private final boolean overrideSecureDecoder;
+
+    public static CodecSelector newInstance(boolean useSecureCodec) {
```

Comment:
`useSecureCodec` seems to mean the opposite of `overrideSecureDecoder` no? 

`useSecureCodec = true` -> "go ahead, use the secure codec"
`overrideSecureCodec = true` -> "override that secure codec, use the insecure one instead"

So... which one is it?